plugins/dashboardmetrics/block.go

```diff
@@ -0,0 +1,205 @@
+package dashboardmetrics
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/iotaledger/hive.go/core/syncutils"
+	"go.uber.org/atomic"
+)
+
+// BlockType defines the component for the different BPS metrics.
+type BlockType byte
```

Comment:
Reuse the type defined in `collector`? They're exactly the same.

Reply:
I assumed that we will remove dashboard soon anyway, so it doesn't matter, but you're right now we can use it